app/models/user.rb

```diff
@@ -6,6 +6,6 @@ class User < ApplicationRecord
   has_many :books
   has_many :bookings
 
-
+  mount_uploader :picture, PhotoUploader
```

Comment:
same comment than for the book model, this should be done through migration @jdharveng

Reply:
Same as above, picture attribute was added through a migration. To mount the uploader needs to be directly on the model as per slides and per documentation here:
https://github.com/carrierwaveuploader/carrierwave